helm/cluster-gcp/templates/_helpers.tpl

```diff
@@ -25,6 +25,7 @@ cluster.x-k8s.io/cluster-name: {{ include "resource.default.name" . | quote }}
 giantswarm.io/cluster: {{ include "resource.default.name" . | quote }}
 giantswarm.io/organization: {{ .Values.organization | quote }}
 helm.sh/chart: {{ include "chart" . | quote }}
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
```

Comment:
Does `index` work with dictionaries? Do you not need to use `get` ? - https://helm.sh/docs/chart_template_guide/function_list/#get